manual/speculative_execution/README.md

```diff
@@ -148,20 +153,15 @@ Cluster cluster = Cluster.builder()
     .build();
 ```
 
-Finally, don't forget to register your tracker with the cluster (the
-policy does not do this itself):
-
-```java
-cluster.register(tracker);
-```
-
-Note that `PerHostPercentileTracker` may also be used with a slow query
+Note that `PercentileTracker` may also be used with a slow query
```

Comment:
In the QueryLogger docs we reference `PerHostPercentileTracker` explicitly, should we change that to `PercentileTracker` too?